### DISCUSSION TITLE
pyproject.toml support

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#8530] [Q&A]
One weblate instance (hosted at Codeberg.org) told me that it is not able to understand `pyproject.toml` file.

Does this depend on the setup specific to that instance or is it a general upstream/weblate problem?

Sidequestion: Why does weblate at all try to read such files. What necessary information is in there about translation?

**Comment by Geeyun-JY3** [comment]
#8537?

**Reply by buhtz** [reply]
On that Issue is also no answer.

It is still unclear if Weblate doesn't support toml or if it is just a setup-issue specific to each installed weblate instance.

**Comment by nijel** [comment]
What is the actual error message you get? I don't see reason why Weblate would read that file unless you are trying to translate it.

**Reply by buhtz** [reply]
That is my point. Then why does Weblate tell me extra that it doesn't support it? :smile:
I will make a screenshot as soon as possible.
